src/main/webapp/placeGuides/PlaceGuideOnList.js

```diff
@@ -418,8 +418,12 @@ class PlaceGuideOnList {
   static close(placeGuideId) {
     const divId = 'placeGuideOnList-' + '{' + placeGuideId + '}';
     const placeGuideDiv = document.getElementById(divId);
-    placeGuideDiv.querySelectorAll('.folded-placeGuide')[0].style.display = 'block';
-    placeGuideDiv.querySelectorAll('.card-placeGuide')[0].style.display = 'none';
-    placeGuideDiv.style.removeProperty('padding');
+    if (placeGuideDiv != null) {
```

Comment:
Should we compare to `undefined` instead? Or is it intentional?

Reply:
It was intentional, because document.getElementById() returns null if the element is not found.